CMakeLists.txt

```diff
@@ -1212,6 +1218,8 @@ add_library(${CoreLibName} ${CoreLinkType}
 	Core/HDRemaster.cpp
 	Core/HDRemaster.h
 	Core/ThreadEventQueue.h
+	Core/WaveFile.cpp
+	Core/Wavefile.h
```

Comment:
If these are always needed, they'll need adding to android as well:

https://github.com/hrydgard/ppsspp/blob/f4e2ca0359e5d2f343cbe8059394ee7b81fd275e/android/jni/Android.mk

-[Unknown]